qemu/tests/cfg/numa_cpu.cfg

```diff
@@ -79,3 +79,54 @@
                   numa_cpu_dieid_cpu4 = 1
                   numa_cpu_coreid_cpu4 = 1
                   numa_cpu_threadid_cpu4 = 1
+      - with_preconfig:
+          qemu_preconfig = on
+          guest_numa_nodes = "node0 node1 node2 node3"
+          mem_devs = "mem0 mem1 mem2 mem3"
+          numa_memdev_node0 = mem-mem0
+          numa_memdev_node1 = mem-mem1
+          numa_memdev_node2 = mem-mem2
+          numa_memdev_node3 = mem-mem3
+          numa_nodeid_node0 = 0
+          numa_nodeid_node1 = 1
+          numa_nodeid_node2 = 2
+          numa_nodeid_node3 = 3
+          size_mem = 1024M
+          # Ensure all node have corresponding cpu in qemu cli
+          guest_numa_cpus = "cpu0 cpu1 cpu2 cpu3"
+          numa_cpu_nodeid_cpu0 = 0
+          numa_cpu_socketid_cpu0 = 0
+          numa_cpu_dieid_cpu0 = 0
+          numa_cpu_coreid_cpu0 = 0
```

Comment:
thread id and socket id are not supported on ppc so we need to skip these parameters on ppc. Thanks.

Reply:
> thread id and socket id are not supported on ppc so we need to skip these parameters on ppc. Thanks.

Configuration for ppc was expected to be defined after L121, let me add x86_64 only for this L84~L119, thanks!

Reply:
Ok, but the job failed on ppc even if there's different config for ppc. Could you re-check it ? 
dcG4kkA,bus=pci.0,addr=0x5  -netdev tap,id=idcG4kkA,vhost=on,vhostfd=20,fd=16  -vnc :0  -rtc base=utc,clock=host  -boot menu=off,order=cdn,once=c,strict=off -enable-kvm"    (status: 1,    output: 'qemu-kvm: -numa cpu,node-id=0,socket-id=0,die-id=0,core-id=0,thread-id=0: thread-id is not supported')

Reply:
Updated, L84~L119 is for x86 only, please help review again, thanks @MiriamDeng